examples/basics/scene/polygon.py

```diff
@@ -0,0 +1,49 @@
+from vispy import app
+import sys
+from vispy.scene import SceneCanvas
+from vispy.scene.visuals import Polygon, Ellipse, Rectangle, RegularPolygon
+from vispy.color import Color
+
+white = Color("#ecf0f1")
+gray = Color("#121212")
+red = Color("#e74c3c")
+blue = Color("#2980b9")
+orange = Color("#e88834")
+
+if __name__ == '__main__':
```

Comment:
Only the last bit should be in this conditional, the rest is fine

Also please add the boilerplate at the top of the file.